packages/@ama-sdk/core/src/plugins/perf-metric/perf-metric.fetch.ts

```diff
@@ -115,15 +162,18 @@ export class PerformanceMetricPlugin implements FetchPlugin {
    * @param response Response of the call associated to the mark to close
    */
   public closeMark(markId: string, response: Response) {
+    const perfMark = this.performance?.mark(this.getPerformanceTag('end', markId)) || undefined;
+    const endTime = perfMark?.startTime ?? this.getTime();
```

Comment:
Dummy question. It is not 'perfMark?.endtime ?? ...'

Reply:
Actually it's a mark so there no "start" and "end", it's just a date when the mark has been created.
The create date of a mark is returned by the `startTime` field.
(cf. https://developer.mozilla.org/en-US/docs/Web/API/PerformanceMark)